spec/app/ics-027-interchain-accounts/README.md

```diff
@@ -104,95 +93,127 @@ function AuthenticateTx(msgs []Any, portId string) error {
-```
-This port will be used to create channels between the controller & host chain for a specific owner/interchain account pair. Only the account with `{owner-address}` matching the bound port will be authorized to send IBC packets over channels created with `ics27-1-{connection-number}-{counterparty-connection-number}-{owner-address}`. The host chain trusts that each controller chain will enforce this port registration and access. 
+ICS-004 allows for each channel version negotiation to be application-specific. In the case of interchain accounts, the channel version set during the `OnChanOpenInit` step (controller chain) must be `ics27-<version>` & the version set during the host chain `OnChanOpenTry` step will include the interchain account address that will be created ([see summary table below](#Version-negotiation-summary)). Note that the generation of this address is stateless, and can be generated in advance of the account creation. 
 
-2. The controller chain emits an event signaling to open a new channel on this port given a connection 
-3. A relayer listening for `OnChannelOpenInit` events will begin the channel creation handshake
-4. During the `OnChanOpenTry` callback on the host chain an interchain account will be registered and a mapping of the interchain account address to the owner account address will be stored in state (this is used for authenticating transactions on the host chain at execution time)
-5. During the `OnChanOpenAck` & `OnChanOpenConfirm` callbacks on the controller & host chains respectively, the `active-channel` for this interchain account/owner pair, is set in state
+Due to how the mechanics of ICS-004 channel version negotiation operate the version passed into the host chain side (OnChanOpenTry) must match the counterparty version passed into the controller side (OnChanOpenAck) otherwise, there will be a resulting error at the IBC protocol level. 
 
+Combined with the one channel per interchain account approach, this method of version negotiation allows us to pass the address of the interchain account back to the controller chain and create a mapping from controller port ID -> interchain account address during the `OnChanOpenAck` callback. As outlined in the [controlling flow](#Controlling-Flow), a controller chain will need to know the address of a registered interchain account in order to send transactions to the account on the host chain.
 
-**Controlling Flow**
+#### Version Negotiation Summary
 
-Once an interchain account is registered on the host chain a controller chain can begin sending instructions (messages) to control this account. 
+| Initiator | Datagram         | Chain acted upon |  Version (Controller, Host) |
+| --------- | ---------------- | ---------------- |  ---------------------- | 
+| Controller| ChanOpenInit     | Controller       |  (ics27-1, none)        | 
+| Relayer   | ChanOpenTry      | Host             |  (ics27-1, ics27-1.{interchain-account-address})        | 
```

Comment:
Define somewhere (either above or below this table), that `interchain-account-address` is the address on the host chain that is controlled by the owner through the given ICA channel